### ISSUE TITLE
wd plugin: error to add warp point

### ISSUE THREAD

**Issue by simonbcn** [oh-my-fish/oh-my-fish#213]
```
 ~/fuentes > wd add fuentes                                                           
fish: Unknown command 'string'
~/.local/share/omf/pkg/wd/functions/__wd_add.fish (line 1): if string match -r "^(\.)+\$" $argv[2] >/dev/null
                                                               ^
in function '__wd_add'
        called on line 103 of file ~/.local/share/omf/pkg/wd/functions/wd.fish
        with parameter list ' fuentes'

in function 'wd'
        called on standard input
        with parameter list 'add fuentes'

fish: Unknown command 'string'
~/.local/share/omf/pkg/wd/functions/__wd_add.fish (line 3):     else if string match "*/*" $argv[2] >/dev/null
                                                                        ^
in function '__wd_add'
        called on line 103 of file ~/.local/share/omf/pkg/wd/functions/wd.fish
        with parameter list ' fuentes'

in function 'wd'
        called on standard input
        with parameter list 'add fuentes'

fish: Unknown command 'string'
~/.local/share/omf/pkg/wd/functions/__wd_add.fish (line 5):     else if string match "* *" $argv[2] >/dev/null
                                                                        ^
in function '__wd_add'
        called on line 103 of file ~/.local/share/omf/pkg/wd/functions/wd.fish
        with parameter list ' fuentes'

in function 'wd'
        called on standard input
        with parameter list 'add fuentes'

fish: Unknown command 'string'
~/.local/share/omf/pkg/wd/functions/__wd_add.fish (line 7):     else if string match "*:*" $argv[2] >/dev/null
                                                                        ^
in function '__wd_add'
        called on line 103 of file ~/.local/share/omf/pkg/wd/functions/wd.fish
        with parameter list ' fuentes'

in function 'wd'
        called on standard input
        with parameter list 'add fuentes'

fish: Unknown command 'string'
in command substitution
        called on line 0 of file ~/.local/share/omf/pkg/wd/functions/__wd_path.fish

in function '__wd_path'
        called on line 2 of file ~/.local/share/omf/pkg/wd/functions/__wd_add.fish
        with parameter list 'fuentes'

in command substitution
        called on line 0 of file ~/.local/share/omf/pkg/wd/functions/__wd_add.fish

in function '__wd_add'
        called on line 103 of file ~/.local/share/omf/pkg/wd/functions/wd.fish
        with parameter list ' fuentes'

in function 'wd'
        called on standard input
        with parameter list 'add fuentes'

cat: /home/juan/.warprc: No existe el fichero o el directorio
Warp point fuentes added
```

**Comment by lfiolhais** [comment]
The string built-in isn't available in fish 2.2.0, it was added afterwards. What is your fish version (`fish --version`)?

**Comment by derekstavis** [comment]
Please report this issue in plugin's repository: https://github.com/fischerling/plugin-wd
